boards/arm/beagleconnect_freedom_cc1352r/beagleconnect_freedom_cc1352r_defconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Yadnik Bendale
```

Comment:
Please credit @statropy per https://github.com/jadonk/beagle_connect_zephyr_board/blob/93a5a3f96b0c8b47634218c9d7a40741459c316d/beagleconnect_freedom_defconfig who did this original work. How did these copyrights get changed?